Felienne/hedy
# [IMPROVEMENT] Always dynamically parse keywords on /explore and program view

**Description**
In this PR we fix an issue where the syntax highlighting was incorrect when viewing a program using non-user keywords. As on the `/explore` page we show all relevant programs instead of only ones with the same language as the user, this inconsistency can be confusing. We solve this my dynamically parse all keywords to the current user language. We also remove the keyword language switcher from the program view page to remove ambiguity.

**Fixes**
This PR fixes #2439

**How to test**
Create a new (or use a second) account and set this to a non-native language, for example Norsk. Create a program, save and share it to make sure it is visible on the `/explore` page. Make sure language specific keywords are used. Then return to your original account and verify that the keywords of the program are dynamically changed to the current user language when viewing and running the program.
